src/components/overlayCookie/overlayCookie.module.scss

```diff
@@ -4,31 +4,12 @@
 
 
 .cookieWrapper {
-  width: 100%;
-  position: fixed;
-  bottom: 0;
   background-color: $color-tertiary;
   opacity: 0.9;
   z-index: index(alert);
```

Comment:
Use the `z-x` utilities in tailwind, maybe we can even define our own instead of the default values, that might makes more sense.